x-pack/packages/kbn-elastic-assistant-common/impl/schemas/conversations/common_attributes.schema.yaml

```diff
@@ -0,0 +1,286 @@
+openapi: 3.0.0
+info:
+  title: Common Conversation Attributes
+  version: 'not applicable'
+paths: {}
+components:
+  x-codegen-enabled: true
+  schemas:
+    NonEmptyString:
+      type: string
+      pattern: ^(?! *$).+$
+      minLength: 1
+      description: A string that is not empty and does not contain only whitespace
+
+    UUID:
+      type: string
+      format: uuid
+      description: A universally unique identifier
+
+    User:
+      type: object
+      description: Could be any string, not necessarily a UUID
+      properties:
+        id:
+          type: string
+          description: User id.
```

Comment:
nit: if `user.name` is guaranteed to be unique, is `id` required?

Reply:
No, I added this just in case :-)